src/gui/systray.cpp

```diff
@@ -101,7 +101,7 @@ Systray::Systray()
 
 #ifdef Q_OS_MACOS
     setUserNotificationCenterDelegate();
-    checkNotificationAuth();
+    checkNotificationAuth(MacNotificationAuthorizationOptions::Default); // No provisional aiuth, ask user explicitly first time
```

Comment:
@claucambra `aiuth` typo in the comment?

Reply:
Nice catch, fixed